src/main/kotlin/io/github/corbym/dokker/dokker.kt

```diff
@@ -10,24 +9,10 @@ fun dokker(init: DokkerContainerBuilder.() -> Unit): DokkerContainer {
     return DokkerContainerBuilder().apply(init).build()
 }
 
-// Search order is
-// 1. process name set in code (if set, DokkerAutoProcessSearchResult.processName is not used)
-// 2. environment variable (if found in path)
-// 3. hardcoded "docker" (if found in path)
-// 3. hardcoded "podman" (if found in path)
 object DokkerAutoProcessSearchResult {
```

Comment:
its not internal anymore?

Reply:
There was some reason I needed to use it in one of the tests to check something.. it might not be required any more icr